[IMP] website: translations (#9945)
odoo/documentation: content/applications/websites/website/configuration/translate.rst

Translated text strings are highlighted in green; Text strings that were not translated
automatically are highlighted in yellow.

.. image:: translate/translated-text.png
  :alt: Entering the translation mode

In this mode, you can only translate texts. To change the page's structure, you must edit the master
page. Each modification on the master page is automatically applied to all translated versions.

.. note::
  When a website supports multiple languages, the content is accessible through different URLs,
  depending on the language selected.
  `https://www.mywebsite.com/shop/product/my-product-1` --> URL to the master page, in English.
  `https://www.mywebsite.com/fr_FR/shop/product/mon-produit-1` --> URL to the French version.

To replace the original text with the translation, click the block, edit its contents, and save.

.. tip::
  Once the desired language is installed, you can translate some items from the backend (e.g., the
  :guilabel:`product’s name` in the product template). To do so, click :guilabel:`EN` next to the
  text you want to translate and add the translation.

Block customization per language

> --------------------------------

Reviewer: - I would not call this section "Block customization" since you can't really customize the block based on the language, you can just hide/show it (like you explained directly below).
- Also, from 17.0, you could just refer to the "Conditional visibility" section in the Building blocks doc and then just keep the last bullet point in your list if you want ("Go to languages to configure ..." - you'll probably need to change it a little bit so it fits the text).

> You can hide content (such as images or videos) that can not be translated. To do so,

Reviewer: ```suggestion
You can hide content (such as images or videos, for example) depending on the language. To do so:
```
it's not just content that cannot be translated, for example, you could decide to hide an entire block just because the content doesn't apply for a language, or any other reason.
